tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "target": "es6",
     "module": "commonjs",
     "lib": [
-      "esnext"
+      "esnext",
+      "DOM"

```

Comment:
Why this is neccesary? It seems incorrect since this is a CLI tool.

Reply:
It is because of this issue https://github.com/axios/axios/issues/4124. It was fixed and merged to the master 8 days ago https://github.com/axios/axios/pull/4229, but it is still not released https://github.com/axios/axios/releases. 

After the release we can upgrade the dependency in https://github.com/Kentico/kontent-core-js/blob/master/package.json